tests/models/test_edition.py

```diff
@@ -131,25 +131,34 @@ def test_missing_coverage_from(self):
             self._db, gutenberg, web)]))
 
     def test_sort_by_priority(self):
-        edition_admin = self._edition(data_source_name=DataSource.LIBRARY_STAFF, with_license_pool=False)
```

Comment:
Apart from the bit at the end, this code change is just cleanup to make the test read easier.